utils/docker/Dockerfile.centos.7

```diff
@@ -67,13 +67,11 @@ RUN yum -y install epel-release; \
 RUN yum -y install \
            python2-avocado python2-avocado-plugins-output-html \
            python2-avocado-plugins-varianter-yaml-to-mux       \
-           yum-plugin-copr python-pathlib                      \
+           python-pathlib                                      \
            ndctl ipmctl e2fsprogs                              \
            python2-clustershell python2-Cython python2-pip     \
            python36-clustershell python36-paramiko             \
            python36-numpy python36-jira python3-pip;           \
-    yum -y copr enable jhli/ipmctl;                            \
-    yum -y copr enable jhli/safeclib;                          \
```

Comment:
Where will these packages come from? They don't appear to be available in EPEL.

Reply:
The current ipmctl package is now only available in EPEL.  The safeclib was a dependency of it.  I have not looked to see if it is no longer a dependency or if it has a new name in EPEL.

The copr library is no longer being referenced by the ipmctl home page.

Reply:
Oh, never mind. I guess they are in EPEL now. Sorry for the noise.